app/src/main/res/layout-ar-land/audio_panel.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layoutDirection="ltr"
```

Comment:
I'll have to test this, but chances are, Amir is correct here. The problem is, layout mirroring enables when you enable the `supportsRtl` flag in the manifest - consequently, the behavior pre-api 17 and post-api 17 differs, especially when you use a phone in the Arabic locale. I ran into this in another app and ended up using this as a fix as well (despite it not being ideal, it was also the best way I could find).